flybywiresim/aircraft
# fix(MCDU): fixes no ALTN handling on windpage and init

Fixes #[issue_no]

## Summary of Changes


This PR fixes the handling of having no alternate defined in a simbrief flight plan. 
Previously the init page would display a NOT IN DATABASE error when importing flight plans without alternate.
The DES wind page would also display NAN values for the alternate wind. 

Now the init page shows no error and the DES wind page doesn't show any alternate information at all according to honeywell manual

## Screenshots (if necessary)



## References





![image](https://github.com/flybywiresim/aircraft/assets/5833783/8bf03835-1fe5-4d11-b06e-003d1f568a01)


## Additional context



Discord username (if different from GitHub): \_chaoz_

## Testing instructions


1. Set up A/C C/D 
2. Power up MCDU
3. Load flight plan without alternate
4. no errors should occur 
5. go to the DES winds page
6. no alternate field should exist
7. no entry into 1R should be accepted
8. Requesting winds should correctly import winds and not show any alternate information

For regression testing doing the same process but with an alternate defined should work as expected


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
